update.go

```diff
@@ -74,6 +74,10 @@ func (b *Bot) ProcessUpdate(u Update) {
 			return
 		}
 
+		if m.Origin != nil && m.Text == "" {
```

Comment:
Why is the text should be empty?

Reply:
It seems to be always empty when forwarding. but if it is for example a reply, the text will not be empty.

Reply:
The emptiness of `m.Text` is already checked here

https://github.com/tucnak/telebot/blob/350ac26ac84216662c19baea128c18fecbc0eb84/update.go#L43

Reply:
Could you please also add the same event for reply to a message? There is a `ReplyTo` field in the `Message`